src/containers/ReviewModal/ResponseContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+    Card
+} from '@edx/paragon';
+
+import createDOMPurify from 'dompurify';
+
+import parse from 'html-react-parser';
+
+import selectors from 'data/selectors';
+
+/**
+ * <ResponseContainer />
+ */
+export const ResponseContainer = ({
+    text
+}) => {
+    return (
+        <Card className="response-card">
+            <Card.Body>
+                {text}
+                {text}
+                {text}
+            </Card.Body>
+        </Card>
+    );
+}
+
+ResponseContainer.defaultProps = {};
+ResponseContainer.propTypes = {
+    text: PropTypes.node.isRequired
+};
+
+const purify = createDOMPurify(window);
+
+export const mapStateToProps = (state) => ({
+    text: parse(purify.sanitize(selectors.submissions.selectedResponse(state)?.text))
```

Comment:
logic should not go in mapStateToProps methods.  this method is for connecting your selectors to components, not for pulling in its own selector logic.

Integrations with external objects/libraries should happen either in a selector, or in the component itself.